### ISSUE TITLE
Bcc recipient doesn't appear in drafts

### ISSUE THREAD

**Issue by pierre-leonard** [thunderbird/thunderbird-android#1857]
### Expected behavior
When opening a draft, the user should be able to see the emails of the bcc field

### Actual behavior
Only the big letter with a coloured background appears instead of the email. Also the cursor position is wrong ( under the background of the letter)

### Steps to reproduce
1. Create a new message
2. Add a bcc address
3. Save as a draft
4. Open the draft and check the bcc field

### Environment
K-9 Mail version: 5.115

Android version:
4.4

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by utk-spartan** [comment]
I would like to work on this issue.

**Comment by philipwhiuk** [comment]
Should hopefully be fairly straightforward. Seems like a sensible beginner issue.

**Comment by utk-spartan** [comment]
I tried to find the root cause of the issue by adding a dummy tokenautocomplete field having only a simple textview as token, and it also bugged out.
The issue happens when objects are added to tokenautocomplete field while the view is being drawn and its visibility is set to GONE, once the view is drawn adding objects or changing its visibility doesn't cause the issue.

I have made a workaround by adding objects only when the view is visible.

Pls review the pull request i have amended the commit
